src/main/java/org/jabref/logic/preferences/SpringerApiKeyPreferences.java

```diff
@@ -0,0 +1,20 @@
+package org.jabref.logic.preferences;
```

Comment:
I think it makes sense to name this preferences CustomAPIKeyPreferences so it can be expanded to include other fetcher keys as well 
@calixtus  What do you think?

Reply:
Me likes it! ;-)

Reply:
Now it is **CustomAPIKeyPreferences**, and an attribute `name` is added.